src/algorithms/calorimetry/ImagingTopoClusterConfig.h

```diff
@@ -22,7 +22,11 @@ struct ImagingTopoClusterConfig {
   // maximum distance of global (x, y) to be considered as neighbors at different layers (if layerMode==xy)
   std::vector<std::variant<std::string, double>> layerDistXY = {1.0 * dd4hep::mm, 1.0 * dd4hep::mm};
   // determines how neighbors are determined for hits in different layers (using either eta and phi, or x and y)
-  enum ELayerMode { etaphi = 0, xy = 1 } layerMode = etaphi;
+  // enum ELayerMode { etaphi = 0, xy = 1 } layerMode = etaphi;
+
+  enum ELayerMode { etaphi = 0, xy = 1, phiz = 2 };
```

Comment:
Probably should make this a [scoped enum](https://en.cppreference.com/w/cpp/language/enum.html) since otherwise it means nobody can use the variable `xy` anywhere (or they have to be careful about shadowing).
```suggestion
  enum class ELayerMode { etaphi = 0, xy = 1, phiz = 2 };
```